src/main/java/frc/robot/commands/AlgaeCommands.java

```diff
@@ -34,7 +34,14 @@ public static Command stopAlgaeGrabber(Subsystems subsystems) {
 
   /** Returns a command that removes algae at the given reef level. */
   public static Command removeAlgaeAtLevel(Subsystems subsystems, ElevatorLevel elevatorLevel) {
-    return Commands.none(); // TODO: implement removeAlgaeAtLevel
+    return Commands.sequence(
+        Commands.runOnce(
+            () -> ElevatorCommands.goToElevatorLevel(subsystems, elevatorLevel),
```

Comment:
The methods of ElevatorCommands return Commands that can be passed directly to Commands.sequence(). Wrapp8ng them in Commands.runOnce() won't wodk.